22-discovery.ks

```diff
@@ -15,6 +15,9 @@ systemctl enable NetworkManager-wait-online.service
 echo " * enabling nm-prepare service"
 systemctl enable nm-prepare.service
 
+echo " * enabling nm-connected service"
+systemctl enable nm-connected.service
+
```

Comment:
Without NetworkManager-wait-online.service we effectively lose the synchronization point in network-online.target. Or do I miss something?